tests/rollen/RolleAnlegen.spec.ts

```diff
@@ -0,0 +1,165 @@
+import { PlaywrightTestArgs, test } from '@playwright/test';
+import { getOrganisationId } from '../../base/api/organisationApi';
+import { createPerson, freshLoginPage, UserInfo } from '../../base/api/personApi';
+import { getRolleId, RollenMerkmal } from '../../base/api/rolleApi';
+import { klasse1Testschule } from '../../base/klassen';
+import { rollenMerkmalLabel } from '../../base/merkmale';
+import { testschuleName } from '../../base/organisation';
+import { rollenArtLabel } from '../../base/rollentypen';
+import { generateNachname, generateVorname } from '../../base/utils/generateTestdata';
+import { PersonManagementViewPage } from '../../pages/admin/personen/PersonManagementView.neu.page';
+import { RolleCreationErrorPage } from '../../pages/admin/rollen/RolleCreationError.page';
+import { RolleCreationSuccessPage } from '../../pages/admin/rollen/RolleCreationSuccess.page';
+import { RolleCreationViewPage } from '../../pages/admin/rollen/RolleCreationView.neu.page';
+import { RolleCreationWorkflow } from '../../pages/admin/rollen/RolleCreationWorkflow.page';
+import { RolleDetailsViewPage } from '../../pages/admin/rollen/RolleDetailsView.neu.page';
+import { RolleManagementViewPage } from '../../pages/admin/rollen/RolleManagementView.neu.page';
+import { HeaderPage } from '../../pages/components/Header.neu.page';
+import { LoginViewPage } from '../../pages/LoginView.neu.page';
+import { StartViewPage } from '../../pages/StartView.neu.page';
+import { rolleCreationParams } from './RolleAnlegen.data';
+
+const ADMIN: string | undefined = process.env.USER;
+const PASSWORD: string | undefined = process.env.PW;
+
+async function setupAndGoToRolleCreationPage(page: PlaywrightTestArgs['page']): Promise<RolleCreationViewPage> {
+  return test.step('Anmelden und zur Rollenanlage navigieren', async () => {
+    const loginPage: LoginViewPage = await freshLoginPage(page);
+    const startPage: StartViewPage = await loginPage.login(ADMIN, PASSWORD);
+    await startPage.waitForPageLoad();
+    const personManagementView: PersonManagementViewPage = await startPage.navigateToAdministration();
+    return personManagementView.menu.navigateToRolleCreation();
+  });
+}
+
+test.describe(`Testfälle für die Rollenanlage: Umgebung: ${process.env.ENV}: URL: ${process.env.FRONTEND_URL}:`, () => {
+  for (const rolleParams of rolleCreationParams) {
+    test.describe(`Rolle der Art ${rolleParams.rollenart} erfolgreich anlegen`, () => {
+      test('Zusammenfassung prüfen', async ({ page }: PlaywrightTestArgs) => {
```

Comment:
```suggestion
      test('Rolle anlegen und Zusammenfassung prüfen', async ({ page }: PlaywrightTestArgs) => {
```
the old text might make sense in the code now but maybe not in the logs later. just trying to make sure we know which test has failed in the future